core/kazoo_amqp/src/api/kapi_presence.erl

```diff
@@ -171,6 +189,23 @@
                      ]).
 -define(SYNC_TYPES, []).
 
+%% Dialog state updates
+-define(DIALOG_HEADERS, [<<"To">>, <<"From">>]).
+-define(OPTIONAL_DIALOG_HEADERS, [<<"Call-ID">>, <<"Direction">>
```

Comment:
in ecallmgr_fs_presence the payload is built with Presence-ID but I don't see it in the list.  I might just be overlooking it...